hmetrics/onload-log/init.go

```diff
@@ -0,0 +1,58 @@
+/* Copyright (c) 2018 Salesforce
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root  or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+/*
+Package onload automatically starts hmetrics reporting, ignoring errors and
```

Comment:
s/onload/onload-log/
s/ignoring errors/logging errors using the stdlib log package/